homework-submission/my-project/src/App.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './App.css';
+import data from './data';
+import Header from './Header';
+import Form from './Form';
+import Todo from './Todo';
+import Edit from './Edit';
+
+class App extends Component {
+  state = {
+    editing: false,
```

Comment:
it is a good idea to great this variable to change the mode to edit or not, BUT:
it is not good to create a variable and  then change the type of it even if the code is working
( it is boolean and then changed to number)

Reply:
you are right, can I use null instead of false ?